scripts/xcode/otd.xclangspec

```diff
@@ -29,6 +60,9 @@
         Syntax = {
             Start = "***";
             End = "***";
+            IncludeRules = (
+				"xcode.lang.url",
```

Comment:
Nit: this line and the next one are using tabs instead of space

Reply:
Nice catch, I will also take the opportunity to format the whole file using _3 spaces indentation_ style.